backend/relayer-archive.md

```diff
@@ -0,0 +1,59 @@
+# Configure relayer "local mode"
```

Comment:
maybe we just add a section on running a Docker container in the readme, instead of these three additional files?